src/components/VPopover/VPopoverContent.vue

```diff
@@ -1,15 +1,20 @@
 <template>
   <div
     v-if="visible"
-    ref="popoverRef"
-    :class="[$style.popover, { [`z-[${zIndex}]`]: zIndex !== undefined }]"
+    class="w-0"
```

Comment:
For whatever reason Safari positions the element very strangely when VoiceOver is open unless using v-show with also the height set to 0. No idea why. It does not appear to effect other browsers I tested.

```suggestion
    v-show="visible"
    class="w-0 h-0"
```

Edit: I'm not sure why this suggestion is rendering so broken. The only changes necessary are to use `v-show` and add `h-0`. The other changes showing in the suggestion please ignore.